Documentation.md

```diff
@@ -0,0 +1,207 @@
+    
+### Timeleft
+
+Currently hardcoded in bot code https://github.com/Rithari/OnePlusBot/blob/master/src/OnePlusBot/Modules/Utility.cs#L369 and disabled.
+Tell how much time lefts before an event when someone queries `;timeleft`.\
+Known issue: it outputs a negative time when event is over.
+
+## Fun
+
+### Starboard
+
+Basically a starboard, users can put a post in a special channel by reacting with :star: emote to it.\
+Minimum amount of stars required to put a post in #starboard channel can be set by users with * by issuing command `;setstars integrer` (eg: `;setstars 5`). Note that higher levels (only configurable in database) are displayed with a different star emote in #starboard channel.
+
+`;starstats` command is usable by everyone and display statistics of starboard posts.\
+Note that system messages (eg: *Username pinned a message*) are ignored by bot and therefore can't go to #starboard channel.
+
+### 8ball
+
+An 8ball command which can be queried by using `;8ball text`.\
+Answers can be "It is certain.", "It is decidedly so.", "Without a doubt.", "Yes, definitely.", "You may rely on it.", "As I see it, yes.", "Most likely.", "Outlook good.", "Yes.", "Signs point to yes.", "Reply hazy to try again.", "Ask again later.", "Better not tell you now.", "Cannot predict now.", "Concentrate and ask again.", "Don't count on it.", "My reply is no.", "Outlook not so good.", "Very doubtful." and "My sources say no."
+
+### Lovecalc
+
+A command which calculates love. It is mostly used between users but can also be with custom text for the first and second parameters.\
+Eg `;lovecalc Username Username2` and `;lovecalc "Pastas" "Margarita pizza"`.
+
+### YouTube search
+
+Search for YouTube videos after `;yt search_terms` command is used and outputs first result as a message in the channel in which one command was executed.
+
+### Urban Dictionary definitions
+
+Returns first result of Urban Dictionary definitions.\
+Syntax is `;define query`\
+It is usable by everyone.
+
+### Steam profile banner displayer
+
+`;steamp`
+
+## Misc
+
+### User avatar display
+
+`;showavatar Username#1234`.
+Also works with user IDs.
+
+### Show bigger versions of emotes
+
+Syntax is `;se :your_emote:`\
+Supports static and animated emotes.\
+It is usable by everyone.
+
+### Echo
+
+This command echo messages.\
+Syntax is `;echo your_message`.\
+It can only be used by users with **Staff** role
+
+### Ping
+
+This command displays the ping between computer on which one bot is running and Discord servers.\
+It can be used by everyone.
+
+### Reminder
+
+Allows you to ask bot to remind you something.\
+Syntax is `;remind lenght your_text` with lenght being days, hours, minutes and seconds.\
+Eg `;remind 2d1h3m write documentation`\
+Bot will send a message in channel in which one you executed command when time will be over with a mention and a jump link to command message.\
+It will also tell you ID of reminder.\
```

Comment:
formulate that a bit better
'After executing the remind command, the bot will ping you and inform you of the id this reminder has' something like that